nss-day-cohort-51/nutshell-vampire-brood
# index.html Changed default Page Title; changed default favicon to vampire icon

# Description: Updates to index.html

- Changed default React Page Title; 
- Changed default favicon to vampire icon

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions

1. Open the application webpage in a browser
2. See the favicon updated to Vampire icon
3. See the Web Page title updates to "**Nutshell...**" 

# Checklist:
